### DISCUSSION TITLE
No validation when adding recipes with API

### DISCUSSION THREAD

**Discussion by stephanvierkant** [mealie-recipes/mealie#3198] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I'm adding some recipes using the API. When adding recipes with incorrect data, it just persist without any errors. When opening the recipe, it gives a 500 error

### Please provide a concise description of the feature that would resolve your issue.

Validation should be added when adding recipes using the API

### Please consider and list out some caveats or tradeoffs made in your design decision

-

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by boc-the-git** [comment]
Please raise a bug report [here](https://github.com/mealie-recipes/mealie/issues/new/choose) and include the relevant information including replication steps.
I can't fully understand the scenario with the information provided.
